iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -0,0 +1,401 @@
+        Objects.requireNonNull(side);
+        switch (side) {
+            case ONE:
+                return checkPermanentLimit1(limitReduction);
+
+            case TWO:
+                return checkPermanentLimit2(limitReduction);
+
+            default:
+                throw new AssertionError("Unexpected side value: " + side);
+        }
+    }
+
+    @Override
+    public boolean checkPermanentLimit(final Side side) {
+        return checkPermanentLimit(side, 1f);
+    }
+
+    @Override
+    public boolean checkPermanentLimit1(final float limitReduction) {
+        return LimitViolationUtils.checkPermanentLimit(this, Side.ONE, limitReduction, getTerminal1().getI());
+    }
+
+    @Override
+    public boolean checkPermanentLimit1() {
+        return checkPermanentLimit1(1f);
+    }
+
+    @Override
+    public boolean checkPermanentLimit2(final float limitReduction) {
+        return LimitViolationUtils.checkPermanentLimit(this, Side.TWO, limitReduction, getTerminal2().getI());
+    }
+
+    @Override
+    public boolean checkPermanentLimit2() {
+        return checkPermanentLimit2(1f);
+    }
+
+    @Override
+    public Overload checkTemporaryLimits(final Side side, final float limitReduction) {
+        Objects.requireNonNull(side);
+        switch (side) {
+            case ONE:
+                return checkTemporaryLimits1(limitReduction);
+
+            case TWO:
+                return checkTemporaryLimits2(limitReduction);
+
+            default:
+                throw new AssertionError("Unexpected side value: " + side);
+        }
+    }
+
+    @Override
+    public Overload checkTemporaryLimits(final Side side) {
+        return checkTemporaryLimits(side, 1f);
+    }
+
+    @Override
+    public Overload checkTemporaryLimits1(final float limitReduction) {
+        return LimitViolationUtils.checkTemporaryLimits(this, Side.ONE, limitReduction, getTerminal1().getI());
+    }
+
+    @Override
+    public Overload checkTemporaryLimits1() {
+        return checkTemporaryLimits1(1f);
+    }
+
+    @Override
+    public Overload checkTemporaryLimits2(final float limitReduction) {
+        return LimitViolationUtils.checkTemporaryLimits(this, Side.TWO, limitReduction, getTerminal2().getI());
+    }
+
+    @Override
+    public Overload checkTemporaryLimits2() {
+        return checkTemporaryLimits2(1f);
+    }
+
+    @Override
+    public List<? extends Terminal> getTerminals() {
+        return Stream.concat(dl1.getTerminals().stream(),
+                             dl2.getTerminals().stream())
+                     .map(index::getTerminal)
+                     .collect(Collectors.toList());
+    }
+
+    @Override
+    public String getName() {
+        return this.id;
+    }
+
+    @Override
+    public boolean hasProperty() {
+        return dl1.hasProperty() || dl2.hasProperty();
+    }
+
+    @Override
+    public boolean hasProperty(final String key) {
+        return dl1.hasProperty(key) || dl2.hasProperty(key);
```

Comment:
Maybe you have to look in the UCTEImporter code. @murgeyseb make a change recently to manage this differently. For instance, what happens if both dangling lines have the property defined, but not equal.